### PR TITLE
docs: Add Data Deal README.MD

### PR DESCRIPTION
**Pull request by inchori**
- Add README for `x/datadeal` module.

### CODE REVIEW

**Review by youngjoon-lee — APPROVED** [review]
👍

File: x/datadeal/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    "...."
+  ]
+}
+```
+
+### Sell Data
+```shell
+SELLER=$(panacead keys show {your seller} -a)
+
+panacead tx datadeal sell-data --data-verification-certificate-file {your data cert json file path} --from $SELLER --chain-id {your chain ID}
```
Comment (inchori) [inline x/datadeal/README.md:27]: How about we rename `--data-verification-certificate-file` to `--data-certificate-file`?

Comment (youngjoon-lee) [reply x/datadeal/README.md:27]: yeah. or, `--data-cert-file`.

Comment (inchori) [reply x/datadeal/README.md:27]: The `--data-cert-file` seems to be good!

**Comment by inchori** [comment]
The `create-deal` and `sell-data` commands will change flag to argument method. This will be implemented in next PR.

**Review by youngjoon-lee — APPROVED** [review]

File: x/datadeal/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+```
+Deal JSON File Example
+```json
+{
+  "data_schema": [
+    "https://www.json.ld"
+  ],
+  "budget": "10000000umed",
+  "max_num_data": 10000,
+  "trusted_data_validator": [
```
Comment (hansol-medi) [inline x/datadeal/README.md:17]:
```suggestion
  "trusted_data_validators": [
```

**Review by hansol-medi — CHANGES_REQUESTED** [review]
Thanks!

File: x/datadeal/README.md (hunk 2 above)
Comment (inchori) [reply x/datadeal/README.md]: Thanks! Fixed in [81e5590](https://github.com/medibloc/panacea-core/pull/333/commits/81e55900ff9d2df340f6736d2a6aa193277cad0b).

**Pull request merged**